insolar/insolar
# INS-3665: add addresses command

**- What I did**
Add command for adding free migration addresses.

**- How to verify it**
Run `./bin/insolar free-migration-count`, check count in shards.
Run `./bin/insolar add-migration-addresses`.
Run `./bin/insolar free-migration-count`, check count in shards.
